lib/meta_tags/controller_helper.rb

```diff
@@ -27,6 +28,19 @@ def render_with_meta_tags(*args, &block)
     end
     protected :render_with_meta_tags
 
+    # Processes meta tags from locales file in the name space
+    # [locale].meta_tags.[controller_name].[action].[title|description|keywords]
+    # So en.meta_tags.visitors.index.title would be loaded as title for default
+    # welcome page in the visitors controller in english.
+    def meta_tags_from_locales
```

Comment:
Assignment Branch Condition size for meta_tags_from_locales is too high. [17.29/15]